gazelle/pythonconfig/pythonconfig.go

```diff
@@ -425,3 +435,19 @@ func (c *Config) SetDefaultVisibility(visibility []string) {
 func (c *Config) DefaultVisibilty() []string {
 	return c.defaultVisibility
 }
+
+// SetTestFilePattern sets the file patterns that should be mapped to 'py_test' rules.
+// Will exit 1 if the pattern is invalid.
+func (c *Config) SetTestFilePattern(patterns []string) {
+	for _, p := range patterns {
+		if !doublestar.ValidatePattern(p) {
+			log.Fatalf("ERROR: Failed to compile glob '%v'. Error: syntax error in pattern\n", p)
```

Comment:
`globstar` errors with [`path.ErrBadPAttern`](https://pkg.go.dev/path/filepath#pkg-variables) but also states "[users should _not_ count on the returned error, `doublestar.ErrBadPattern`, being equal to `path.ErrBadPattern`](https://pkg.go.dev/github.com/bmatcuk/doublestar/v4#Match)" so I've opted to inject the string manually.

Reply:
Actually... I could just do:

```go
// We only care if the pattern is valid. Use `Match` instead of `ValidatePattern` (which
// just returns `bool`) so that we always use the error message provided by `doublestar`.
if _, err := doublestar.Match(p, ""); err != nil {
    log.Fatalf("ERROR: Failed to compile glob '%v'. Error: %v\n", p, err)
}
...
```

Any preference between
(a) using `ValidatePattern` and manually injecting the error string as I've already done or
(b) abusing `Match`?

Reply:
I guess the word `abusing` in your comment says for itself :) Looking at the `Match`'s code, `err.ErrBadPattern` brings no more than just an error flag (no error message, no extra context etc.) so it is effectively does the same as having true/false. I'd keep it as is. Except maybe a nit regarding go error message [style](https://go.dev/wiki/CodeReviewComments#error-strings) (see the comment in `configure.go`).

```suggestion
			log.Fatalf("invalid glob pattern '%s'", p)
```

Reply:
👍  kept, aside from style nit